spanner/api/Spanner.Samples.Tests/QueryDataWithNumericParameterTest.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class QueryDataWithNumericParameterTest
+{
+    private readonly SpannerFixture _spannerFixture;
+
+    public QueryDataWithNumericParameterTest(SpannerFixture spannerFixture)
+    {
+        _spannerFixture = spannerFixture;
+    }
+
+    [Fact]
+    public async Task TestQueryDataWithNumericParameter()
+    {
+        await _spannerFixture.CreateVenuesTableAndInsertDataAsync();
```

Comment:
To give some background, in other languages, we still have the datatypes samples so the NUMERIC samples follow on from that. But we decided to remove it in C# since those samples weren't pulled into the docs. The downside of it is that we have to do this extra setup step.